### DISCUSSION TITLE
How do I suggest a new feature for Docker Extensions?

### DISCUSSION THREAD

**Discussion by amyb12345** [docker/extension-ideas#56] [Announcements]
If you'd like to suggest a new feature for the Extensions, please add it to [Docker's public roadmap](https://github.com/docker/roadmap/projects/1).

If your feature request is for a specific extension that you are using, check out the extensions detail page in the Extensions Marketplace. 
1. Click Add Extensions
2. Find the extension and click on the extension card.
3. Scroll to the bottom of the Extension detail page. 

Many extension authors provide links for support and feature requests.

Docker does not build or maintain the majority of extensions in the Extensions Marketplace. It's best to work directly with the publisher so that they can understand your use case.

**Comment by vjkrishn** [comment]
Energy Modeling software

**Reply by ivan-curkovic** [reply]
Hi @vjkrishn, do you have any examples? What kind of software are you using?
